core/src/main/java/org/testcontainers/images/TimeLimitedLoggedPullImageResultCallback.java

```diff
@@ -19,7 +18,7 @@
 import static org.testcontainers.DockerClientFactory.TESTCONTAINERS_THREAD_GROUP;
 
 /**
- * {@link PullImageResultCallback} with improved logging of pull progress and a 'watchdog' which will abort the pull
+ * {@link LoggedPullImageResultCallback} with improved logging of pull progress and a 'watchdog' which will abort the pull
```

Comment:
Not a big deal, but I'd query this change..

Reply:
Shouldn't this instead be a reference to `com.github.dockerjava.api.command.PullImageResultCallback`, which is the root-iest subclass that this class is a specialisation of?

From the perspective of someone looking at the public APIs, `LoggedPullImageResultCallback` doesn't exist.

Reply:
@rnorth I wanted to avoid referencing 3rd party classes at all, but don't mind changing it to `PullImageResultCallback` :)